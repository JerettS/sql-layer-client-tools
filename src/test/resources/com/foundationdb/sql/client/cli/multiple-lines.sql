SELECT 'a',
       'b',
       'c';
SELECT 1; SELECT 2; SELECT 4,
                           5;
SELECT 6; SELECT
                 7;
S
E
L
E
C
T
 8
;
