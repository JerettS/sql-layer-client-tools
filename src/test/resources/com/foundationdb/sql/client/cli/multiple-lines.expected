 _SQL_COL_1 | _SQL_COL_2 | _SQL_COL_3 
------------+------------+------------
 a          | b          | c          
(1 row)

 _SQL_COL_1 
------------
          1 
(1 row)

 _SQL_COL_1 
------------
          2 
(1 row)

 _SQL_COL_1 | _SQL_COL_2 
------------+------------
          4 |          5 
(1 row)

 _SQL_COL_1 
------------
          6 
(1 row)

 _SQL_COL_1 
------------
          7 
(1 row)

 _SQL_COL_1 
------------
          8 
(1 row)

